modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -25,6 +29,21 @@ def validate_form_526_claimant_certification!
         raise ::Common::Exceptions::InvalidFieldValue.new('claimantCertification',
                                                           form_attributes['claimantCertification'])
       end
+
```

Comment:
I know this is the pattern in v1, but do we need a method to call a method? Could we remove lines 33-35, and put 'palidate_form_526_current_mailing_address_country!' in the main method `validate_form_526_submission_values!`?

Reply:
All set.